index.html

```diff
@@ -3,18 +3,15 @@
 <head>
   <link rel="stylesheet" href="styles.css">
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=<device-width>, initial-scale=1.0">
-  <link rel="preconnect" href="https://fonts.googleapis.com">
-  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  
   <title>Luis's Portfolio</title>
 </head>
 <body>
   <header>
-    <div id="welcome">Welcome</div>
+    <div id="welcome"><a>Welcome</a></div>
```

Comment:
- [ ] As stated by the previous reviewer there is a horizontal scroll bar that is making your website look unprofessional, and there is also an empty space on the right side of the page 🔽 , you can fix both issues by setting the `box-sizing` property and resetting margins and paddings to all elements of your page.

![2023-03-25_023234](https://user-images.githubusercontent.com/104319462/227682799-0a282b81-8805-4b49-975f-6003dcccca3b.png)

Reply:
Box sizing removed scrolling. Thanks. But despite i removed all margin and padding there is still a blank space.

Reply:
Make sure to use the "all" selector like this 🔽 
```
* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}
```
This should fix the problem

Reply:
Thanks a lot.